src/application.coffee

```diff
@@ -150,8 +150,12 @@ fetch = (app, setDeviceUpdateState = true) ->
 		device.updateState(status: 'Downloading', download_progress: 0)
 
 		Promise.try ->
-			JSON.parse(app.env)
-		.then (env) ->
+			# In dependent apps env is replaced with config
```

Comment:
If this is coming from `parseEnvAndFormatRemoteApps` then `app.env` won't include `RESIN_SUPERVISOR_DELTA`?

I'm ok in general with shortcuts like this if they're necessary to get it ready a bit sooner, but I would like them to be cleaned up shortly after, rather than hanging around forever (since the supervisor code is already fairly complex)